core-chart/templates/postcreatehooks/wds.yaml

```diff
@@ -43,185 +38,132 @@ spec:
+            image: quay.io/kubestellar/kubectl:{{.Values.KUBECTL_VERSION}}
+            imagePullPolicy: IfNotPresent
+            command: [ "bin/sh", "-c", "sh /mnt/config/get-kubeconfig.sh '{{.ITSName}}' true | base64 -d > /mnt/shared/transport-kubeconfig" ]
+            volumeMounts:
+            - name: config-volume
+              mountPath: /mnt/config
+            - name: shared-volume
+              mountPath: /mnt/shared
+          containers:
+          - name: transport-controller
+            image: ghcr.io/kubestellar/kubestellar/ocm-transport-controller:{{.Values.TRANSPORT_VERSION}}
             imagePullPolicy: IfNotPresent
             args:
-            - upgrade
-            - --install
-            - --namespace
-            - "{{"{{.Namespace}}"}}"
-            - ocm-transport-controller
-            - oci://ghcr.io/kubestellar/kubestellar/ocm-transport-controller-chart
-            - --version
-            - {{.Values.KUBESTELLAR_VERSION}}
-            - --set
-            - "{{"wds_cp_name={{.ControlPlaneName}}"}}"
-            - --set
-            - "{{"transport_cp_name={{.ITSName}}"}}"
-            - --set
-            - "verbosity={{.Values.verbosity.transport | default .Values.verbosity.default | default 4 }}"
-            - --set
-            - "max_num_wrapped={{.Values.transport_controller.max_num_wrapped}}"
-            - --set
-            - "max_size_wrapped={{.Values.transport_controller.max_size_wrapped}}"
-            env:
-            - name: XDG_CACHE_HOME
-              value: /tmp/helm/.cache
-          restartPolicy: Never
-      backoffLimit: 1
+            - --metrics-bind-address={{.Values.transport_controller.metrics_bind_addr}}
+            - --pprof-bind-address={{.Values.transport_controller.pprof_bind_addr}}
+            - --transport-kubeconfig=/mnt/shared/transport-kubeconfig
+            - --transport-qps={{.Values.transport_controller.transport_qps}}
+            - --transport-burst={{.Values.transport_controller.transport_burst}}
+            - --wds-kubeconfig=/mnt/shared/wds-kubeconfig
+            - --wds-name={{"{{.ControlPlaneName}}"}}
+            - --wds-qps={{.Values.transport_controller.wds_qps}}
+            - --wds-burst={{.Values.transport_controller.wds_burst}}
+            - -v={{.Values.verbosity.transport | default .Values.verbosity.default | default 4 }}
+            - --max-num-wrapped={{.Values.transport_controller.max_num_wrapped}}
+            - --max-size-wrapped={{.Values.transport_controller.max_size_wrapped}}
+            volumeMounts:
+            - name: shared-volume
+              mountPath: /mnt/shared
+              readOnly: true
+          volumes:
+          - name: shared-volume
+            emptyDir: {}
+          - name: config-volume
+            configMap:
+              name: transport-controller-config
+              defaultMode: 0744
+  # ^^^^ TRANSPORT CONTROLLER ^^^^
+
+  # vvvv KUBESTELLAR CONTROLLER vvvv
+  # KUBESTELLAR_CONTROLLER_PLACEHOLDER

```

Comment:
I do not see the `# ` in the stuff being replaced in the `sed` command in the Makefile.

Reply:
The `#` does not matter because the entire line is replaced